tests/Client/GuzzleClientTest.php

```diff
@@ -34,6 +34,14 @@
 
 class GuzzleClientTest extends TestCase
 {
+    protected function setUp(): void
+    {
+        // Skip tests if GuzzleHttp\Client is not installed
+        if (!class_exists('GuzzleHttp\Client')) {
```

Comment:
OK but then it means that we actually never test the guzzle client right? 🤔
Maybe we should still keep guzzle in the require-dev in composer so that we can still run tests in the CI